registry/remote/auth/example_test.go

```diff
@@ -199,41 +265,23 @@ func ExampleClient_Do_withAccessToken() {
 	// 200
 }
 
-// ExampleClient_Do_clientConfiguration shows the client configurations available,
-// including using cache, setting user agent and configuring OAuth2.
-func ExampleClient_Do_clientConfiguration() {
+func ExampleClient_Do_withRefreshToken() {
```

Comment:
Doc: we still need commentary doc for this func

Reply:
Added the commentary doc.